package NKikimrStorageSettings;
option java_package = "ru.yandex.kikimr.proto";

message TChannelPurpose {
    enum EPurpose {
        SysLog   = 1;
        Log      = 2;
        Data     = 3;
        External = 4;
    };
    optional EPurpose Purpose = 1;
    optional uint32 Channel = 2;
}

message TStorageRoom {
    optional uint32 RoomId = 1;
    repeated TChannelPurpose Explanation = 2 [deprecated = true];
    repeated TChannelPurpose Channels = 3;
}
